tests/Benchmarks/System.Buffers/Reader.cs

```diff
@@ -45,7 +45,7 @@ public void ParseInt32BufferReader()
         {
             var reader = BufferReader.Create(s_ros);
 
-            while (BufferReaderExtensions.TryParse(ref reader, out int value))
+            while (reader.TryParse(out int value))
```

Comment:
Why are we advancing by 1 each time? Shouldn't it be 4 since we have "1234 1234 1234 ..."?